{{cookiecutter.project_slug}}/Makefile

```diff
@@ -4,6 +4,8 @@ NOTEBOOKS_PATH?=notebooks
 REQUIREMENTS_PIP?=requirements.txt
 REQUIREMENTS_APT?=apt.txt
 RESULTS_PATH?=results
+PROJECT_PYTHON_FILES=setup.py setup.cfg README.md LICENSE
```

Comment:
Why do you need to copy README.md and LICENCE? I can't imagine a use-case when having any of these two in the storage may be useful.

Reply:
My logic was that the template should create an environment as much close to the local environment as possible. However, I agree, README.md and LICENSE can be deleted.